CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: A new experimental option, Inverted Aether. In this mode, it's the Light Aether atmosphere that is dangerous! All safe zones are moved to Light Aether, but that's not enough so it's still extremely dangerous. This mode has no logic.
 
+#### Logic Database
+
+- Added: More options to reach the item in Communication Area.
```

Comment:
Please be explicit on what these alternatives were.
We use the term "pickup" in this case, not item.